src/gbyte.f

```diff
@@ -1,36 +1,34 @@
 C> @file
-C> THIS IS THE FORTRAN VERSION OF GBYTE      
-C      
-C> THIS IS THE FORTRAN VERSION OF GBYTE
-C> @author DR. ROBERT C. GAMMILL, CONSULTANT, NATIONAL CENTER FOR ATMOSPHERIC RESEARCH
-C> @date MAY 1972
+C> @brief This is the fortran version of gbyte.
+C> @author Dr. Robert C. Gammill @date 1972-05-DD
+
+C> This is the fortran version of gbyte
 C>
-C> CHANGES FOR SiliconGraphics IRIS-4D/25
-C> SiliconGraphics 3.3 FORTRAN 77
-C> March 1991, RUSSELL E. JONES
-C> NATIONAL WEATHER SERVICE
+C> Program history log:
+C> - Russell E. Jones 1991-03-DD
+C> Changes for SiliconGraphics IRIS-4D/25
+C> SiliconGraphics 3.3 FORTRAN 77.
 C>
-C> SUBROUTINE GBYTE (IPACKD,IUNPKD,NOFF,NBITS)
+C> To unpack a byte into a target word.  the
+C> unpacked byte is right-justified in the
+C> target word, and the remainder of the
+C> word is zero-filled.
 C>
-C> TO UNPACK A BYTE INTO A TARGET WORD.  THE
-C> UNPACKED BYTE IS RIGHT-JUSTIFIED IN THE
-C> TARGET WORD, AND THE REMAINDER OF THE
-C> WORD IS ZERO-FILLED.
+C> @param[in] IPACKD The word or array containing the byte to be
+C>                   unpacked.
 C>
-C> @param[in] IPACKD THE WORD OR ARRAY CONTAINING THE BYTE TO BE
-C>                   UNPACKED.
+C> @param[out] IUNPKD The word which will contain the unpacked
+C>                   byte.
```

Comment:
Remove the extra whitespace before "byte", and do the same for all the other lines of documentation. One space is all that is ever needed.